lib/paper_trail/record_trail.rb

```diff
@@ -563,17 +570,16 @@ def attribute_changed_in_latest_version?(attr_name)
     # Rails 5.1 changed the API of `ActiveRecord::Dirty`. See
     # https://github.com/airblade/paper_trail/pull/899
     #
-    # Event can be any of the three (create, update, destroy).
+    # Event can be any of the following (create, update, destroy, touch).
```

Comment:
"touch" is not an event. There are still only three events: create, update, and destroy. `touch` will cause the insertion of an "update" event. See `record_update`.

I think this will be confusing to many people, and I'm wondering how we can document it better.

Reply:
Well should we make `touch` an event? I would simply add another optional argument to `record_update(force:, in_after_callback:, event: "update")` or add a `record_touch` method.

Otherwise I can simply add documentation in the readme next to the touch documentation stating that the event will be recorded as `update`

Reply:
> should we make touch an event?

I don't know. 

Reasons against: A `touch` is simply an update. It updates timestamps. If we had a touch event, the lifecycle would be harder to understand. Instead of eg. create-update-update-destroy, it's eg. create-update-touch-touch-update-touch-destroy. The "middle part" of the lifecycle becomes heterogeneous. This heterogeneity would be a major breaking change.

Reasons for: Maintains a one-to-one correspondence between callbacks and events, which makes callbacks easier to understand I guess. It's not a big advantage.

So I guess I'm leaning against making touch an event.

Reply:
Okay then in that case, I simply updated the readme stating the event will be saved as `update`